src/applications/static-pages/static-pages-entry.js

```diff
@@ -100,6 +100,7 @@ import createVYEEnrollmentWidget from './vye-enrollment-login-widget/createVYEEn
 import create1010DAccess from './ivc-champva/10-10D/entry';
 import create107959CAccess from './ivc-champva/10-7959c/entry';
 import create107959F1Access from './ivc-champva/10-7959f-1/entry';
+import createMhvSigninCallToAction from './mhv-signin-cta/createMhvSigninCTA';
```

Comment:
Note: I will not be fixing any of these cross app imports as it is beyond the scope of this issue.